typing_extensions/src_py2/typing_extensions.py

```diff
@@ -89,3 +95,259 @@ def __subclasshook__(cls, C):
                     return True
             return NotImplemented
 
+
+def _collection_protocol(cls):
+    # Selected set of collections ABCs that are considered protocols.
+    name = cls.__name__
+    return (name in ('ABC', 'Callable', 'Awaitable',
```

Comment:
I don't think this will cause problems, but just for consistency I will remove this, thus prohibiting to create subprotocols of ``Iterable`` etc. Anyway, there is a separate ``typing`` PR #417 that is oriented for post-typeshed-PR time.